jac/jaclang/compiler/passes/main/import_pass.py

```diff
@@ -463,27 +464,38 @@ def __load_builtins(self) -> None:
         from jaclang.compiler.passes.main import PyastBuildPass
 
         assert isinstance(self.ir, ast.Module)
-
-        file_to_raise = str(
-            pathlib.Path(os.path.dirname(__file__)).parent.parent.parent
-            / "vendor"
-            / "mypy"
-            / "typeshed"
-            / "stdlib"
-            / "builtins.pyi"
+        base_dir = pathlib.Path(os.path.dirname(__file__)).parent.parent.parent
+        jac_gen_dir = (
+            base_dir / "vendor" / "mypy" / "typeshed" / "stdlib" / "__jac__gen__"
         )
-        with open(file_to_raise, "r", encoding="utf-8") as f:
-            file_source = f.read()
-            mod = PyastBuildPass(
-                input_ir=ast.PythonModuleAst(
-                    py_ast.parse(file_source),
-                    orig_src=ast.JacSource(file_source, file_to_raise),
-                ),
-            ).ir
-            mod.parent = self.ir
-            SubNodeTabPass(input_ir=mod, prior=self)
-            SymTabBuildPass(input_ir=mod, prior=self)
-            mod.parent = None
+        mod_file_path = jac_gen_dir / "builtins_mod.jbc"
```

Comment:
Why the extension of the file is called `.jbc` as it shows it might be a jac byte code file but it's a pickel dump? shouldn't it be a `.pkl` or `.dump` file?